src/backend/core/tests/test_models_invitations.py

```diff
@@ -138,3 +141,64 @@ def test_models_team_invitations_get_abilities_member():
         "patch": False,
         "put": False,
     }
+
+
+def test_models_team_invitations_email():
+    """Check email invitation during invitation creation."""
+
+    member_access = factories.TeamAccessFactory(role="member")
+    team = member_access.team
+
+    # pylint: disable-next=no-member
+    assert len(mail.outbox) == 0
+
+    factories.TeamAccessFactory(team=team)
+    invitation = factories.InvitationFactory(team=team, email="john@people.com")
+
+    # pylint: disable-next=no-member
+    assert len(mail.outbox) == 1
+
+    # pylint: disable-next=no-member
+    (email,) = mail.outbox
+
+    assert email.to == [invitation.email]
+    assert email.subject == "Invitation to join Desk!"
+
+    email_content = " ".join(email.body.split())
+    assert "Invitation to join Desk!" in email_content
+
```

Comment:
if we decide to mention invitation.team in the email, we should check invitation.issuer and invitation.email appear correctly in the content.

Reply:
Ah, for now I'll stick to Johann design. The more important part is t ship smth, we'll iterate on the content itself later on.